opre_ops/ops_site/migrations/0007_canamount_caninfo.py

```diff
@@ -0,0 +1,47 @@
+# Generated by Django 3.2.6 on 2021-09-17 21:45
+
+from django.db import migrations, models
+import django.db.models.deletion
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('ops_site', '0006_alter_person_division'),
+    ]
+
+    operations = [
+        migrations.CreateModel(
+            name='CANInfo',
+            fields=[
+                ('id', models.BigAutoField(auto_created=True, primary_key=True, serialize=False, verbose_name='ID')),
+                ('number', models.CharField(max_length=30)),
+                ('description', models.CharField(max_length=100)),
+                ('nickname', models.CharField(max_length=30)),
+                ('purpose', models.TextField(null=True)),
```

Comment:
[_issue, blocking_]: Following up on our conversation in Slack, suggest that we use `blank=True` here instead. This should resolve the issue we saw where CANs fail to save in Django Admin.

https://docs.djangoproject.com/en/3.2/ref/models/fields/#null
https://docs.djangoproject.com/en/3.2/ref/models/fields/#blank

Reply:
@alexsoble I think this is resolved now, can you double check

Reply:
Yep, this looks resolved when I try editing/saving CANs locally!